contrib/scripts/checkpr.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
Could probably add a simple comment below that the script requires curl, jq, and paplay (if on Linux). Could do it as a comment, or an actual check something like:

```
if ! which curl >/dev/null || ! which jq >/dev/null; then
    echo "This tool requires 'curl' and 'jq' utilities to work correctly. Please install them."
    exit 1
fi
```